tests/receivers/prometheus/testdata/resource_metrics/internal.yaml

```diff
@@ -4,6 +4,9 @@ resource_metrics:
       net.host.port: "8889"
       service.instance.id: localhost:8889
       service.name: otel-agent
+      service_instance_id: <ANY>
+      service_name: otelcol
+      service_version: <VERSION_FROM_BUILD>
```

Comment:
These are added to resource attributes due to `target_info` metric now being exposed. The prometheus receiver adds the [labels on target_info to resource attrs](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/prometheusreceiver/internal/transaction.go#L323).

Reply:
I think that's ok. Ideally this should've been configurable on the Prometheus receiver side

Reply:
Yes, just wanted to note that all internal metrics have 2 sets of `service name`, and `service instance id`. The [ones added by the receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/prometheusreceiver/internal/prom_to_otlp.go#L42-L48) in otel format and a 2nd set from target_info with different values, which might become confusing.

Reply:
@jinja2 do they have different keys with the same values?

Reply:
Are you asking about attributes from target_info and the ones added by the receiver, right? I don't think I have seen any in the tests I have worked on.